staging/src/k8s.io/kubectl/pkg/polymorphichelpers/logsforobject_test.go

```diff
@@ -139,6 +139,70 @@ func TestLogsForObject(t *testing.T) {
 				FieldPath:  fmt.Sprintf("spec.containers{%s}", testPodWithOneContainers().Spec.Containers[0].Name),
 			}},
 		},
+		{
+			name: "pods list logs: different containers",
```

Comment:
Let's clarify this test name like the release note

```suggestion
			name: "pods list logs: default container should not leak across pods",
```